README.md

```diff
@@ -285,7 +285,15 @@ You should generally not run the agent as `root` unless you can't use
 capabilities for some reason.
 
 #### Windows
-On Windows the agent must be installed and run under an administrator account.
+On Windows, the Smart Agent can be installed and run using an Administrator account. You can also run the Smart Agent in non-Administrator mode.
+
+There are some monitors and observers that require special permissions, as noted below.
```

Comment:
@jcheng-splunk Hi, Jeff. I don't see a section called "Configure user privileges" on https://github.com/signalfx/signalfx-agent/blob/main/docs/agent-install-standalone-windows.md. Is there another page that has this section?

Reply:
@tmorrisonsfx the ask is instead of mentioning permissions in `README.md` file should we have to provide a direct link to the `Configure user privileges` section of `agent-install-standalone-windows.md` that I have added in this PR? or is it ok to mention in both file?

Reply:
Oh, I see. Yes, I think it's ok to provide a direct link. Thanks.

Reply:
Hi @tmorrisonsfx @jcheng-splunk,
I have updated the `README.md` file accordingly. Can you please review it?